src/Uno.UI/Microsoft/UI/Xaml/Controls/AppBar/AppBarAutomationPeer.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.UI.Xaml.Controls;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	public partial class AppBarAutomationPeer : global::Windows.UI.Xaml.Automation.Peers.FrameworkElementAutomationPeer, global::Windows.UI.Xaml.Automation.Provider.IToggleProvider, global::Windows.UI.Xaml.Automation.Provider.IExpandCollapseProvider, global::Windows.UI.Xaml.Automation.Provider.IWindowProvider
```

Comment:
The `global::` are probably not necessary, add `using` statements for those namespaces to make it cleaner 🙂